website/api_versioned_docs/version-1.0/zkvm/quickstart.md

```diff
@@ -80,7 +80,7 @@ Note that since proofs are now being generated, the execution time will be
 significantly longer than when running in dev-mode. To create a proof with the
 zkVM on your own machine, we recommend at least 16 GB of RAM. To avoid these
 hardware requirements, consider using [Bonsai] to generate proofs remotely, as
-it will be significantly faster than running proofs locally. You can [request
+it will be significantly faster than running proofs locally. You can \[request
```

Comment:
this breaks the link: https://docs-website-ogezjx410-risczero.vercel.app/api/zkvm/quickstart#5-real-proof-generation